integ-tests/hadoop/src/main/java/com/google/cloud/bigquery/dwhassessment/testdata/TestDataHelper.java

```diff
@@ -52,32 +53,55 @@ private TestDataHelper() {}
    */
   public static void generateDbWithTables(Connection connection, int tableCount)
       throws SQLException, IOException {
-    String createDb = SqlHelper.getSql(Constants.SQL_TEST_DATA_BASE_PATH + "create_db.sql");
-    String createTable = SqlHelper.getSql(Constants.SQL_TEST_DATA_BASE_PATH + "create_table.sql");
+    String createDb = SqlHelper.getSql(SQL_TEST_DATA_BASE_PATH + "create_db.sql");
+    String createTable = SqlHelper.getSql(SQL_TEST_DATA_BASE_PATH + "create_table.sql");
+    String populateData = SqlHelper.getSql(SQL_TEST_DATA_BASE_PATH + "populate_data.sql");
```

Comment:
So is this a format string? If so, maybe rename it accordingly.

Reply:
No, it isn't format string getSql method reads content of sql files

Reply:
Ah, yes, I got confused by the usage of format but I now remember that the SQL is really a format string that gets rendered to SQL. That's alright then.